spikes/docker-postgres/README.md

```diff
@@ -0,0 +1,95 @@
+# Docker Postgres Setup
+
+Basic dockerized `postgres` and `pgadmin` services with room to grow.
+
+Requires [Docker](https://www.docker.com/products/docker-desktop) and Docker Compose (comes with a standard Docker Desktop installation for Mac and Windows.)
+
+## Running Locally
+
+From your favorite Command Line Interface (e.g. `bash` or `powershell`), enter the command:
+
+```bash
+docker-compose up
+```
+
+This downloads and builds the base Docker images (if needed), and starts both services.
+
+`pgadmin` can now be accessed from http://localhost:8080.
+
+The `postgres` service exposes the standard PostgreSQL port `5432`.
+
+Alternatively (e.g. if one service or another is already started):
+
+```bash
+docker-compose up [SERVICE]
+```
+
+Where `[SERVICE]` is `postgres` or `pgadmin`.
+
+To shut everything down:
+
+```bash
+docker-compose down
```

Comment:
I suggest using `docker-compose stop` as the default shutdown command and noting `docker-compose down` will destroy data and any changes made in postgres.

Reply:
Great point @tienyuan, thanks for catching this. Updates made, calling out specifically the destructive nature of `docker-compose down`.